### PR TITLE
minor adjustments

### PR DESCRIPTION
**Pull request by BleakleyMcD**

### CODE REVIEW

File: writelto [hunk 1]
```diff
@@ -20,6 +19,8 @@ else
     VERSION=""
 fi
 
+LTO_LOGS="${LTO_INDEX_DIR}"
```
Comment (retokromer) [inline writelto]: Why here and not with the other variables?

I suggest to move the
```sh
 . "${SCRIPTDIR}/mmfunctions" || { echo "Missing '${SCRIPTDIR}/mmfunctions'. Exiting." ; exit 1 ; }
```
before the variable definitions.

File: indexschemas [hunk 2]
```diff
@@ -3,7 +3,7 @@
 # indexschemas
 # This script makes a text file listing certain files from the ltfs schema file.
 
-LTO_LOGS="${HOME}/Documents/lto_indexes"
+LTO_LOGS="${LTO_INDEX_DIR}"
```
Comment (dericed) [inline indexschemas]: perhaps if LTO_INDEX_DIR is not set, then it should stop or use a fallback. I prefer to use "${HOME}/Documents/lto_indexes" as the fallback (reverse compatible) with a warning.

Comment (retokromer) [inline indexschemas]: Good point!

**Comment by BleakleyMcD** [comment]
fixed. i think.

File: writelto [hunk 3]
```diff
@@ -4,22 +4,23 @@
...
-. "${SCRIPTDIR}/mmfunctions" || { echo "Missing '${SCRIPTDIR}/mmfunctions'. Exiting." ; exit 1 ; }
+_check_for_lto_index_dir
 
 if [[ $(dirname $(which "${0}")) = "/usr/local/bin" || $(dirname $(which "${0}")) = "${HOME}/.linuxbrew/bin" ]] ; then
     VERSION=$(TMP=$(brew info ltopers | grep -Eo "/ltopers/.* \(") ; echo ${TMP:9:(${#TMP}-11)})
 else
     VERSION=""
 fi
 
+
```
Comment (retokromer) [inline writelto]: Could you possibly delete one of the two empty lines? Thank you!

File: indexschemas (hunk 2 above)
Comment (retokromer) [inline indexschemas]: Resolved in https://github.com/mediamicroservices/mm/pull/170.

**Pull request merged**